tests/app/celery/v3/test_notification_tasks.py

```diff
@@ -290,3 +303,31 @@ def test_v3_send_sms_notification(notify_db_session, mocker, sample_notification
     assert notification.status == NOTIFICATION_SENT
     assert notification.reference == "provider reference"
     assert notification.sent_by == "client name"
+
+
+def test_v3_process_sms_notification_with_noexistent_template(
```

Comment:
can the typo in this method be easily resolved? should be `test_v3_process_sms_notification_with_nonexistent_template`

Reply:
done